account/src/main/resources/org/killbill/billing/account/ddl.sql

```diff
@@ -23,6 +23,7 @@ CREATE TABLE accounts (
     country varchar(50) DEFAULT NULL,
     postal_code varchar(16) DEFAULT NULL,
     phone varchar(25) DEFAULT NULL,
+    notes varchar(4096) DEFAULT NULL,
```

Comment:
Could you also provide a migration file?

See http://docs.killbill.io/0.16/database_migrations.html (see https://github.com/killbill/killbill/blob/work-for-release-0.17.x/invoice/src/main/resources/org/killbill/billing/invoice/migration/V20163917173937__invoice_status_459.sql for example).